x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/SecurityFeatureSetUsage.java

```diff
@@ -69,6 +80,8 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeMap(realmsUsage);
         out.writeMap(rolesStoreUsage);
         out.writeMap(sslUsage);
+        out.writeMap(tokenServiceUsage);
```

Comment:
@bizybot I missed this in my review but there is a bug here; we write the map always without checking the version. We need the same guards on both reading and writing